ONSdigital/ras-frontstage
# remove ignore

# What and why?
This PR removes an unneeded ignore from the makefile.

# How to test?
Deploy and run tests.

Do a pipenv install, pipenv graph, and check that the following depenencies have updated:
certifi = ">=2024.07.04"
requests = ">=2.32.0"
urllib3 = ">=2.2.2"

# Jira
[Card](https://jira.ons.gov.uk/browse/RAS-1315)
